### PR TITLE
Add Dockovpn to VPN list

### PR DESCRIPTION
**Pull request by alekslitvinenk**
### Dockovpn / VPN
[Dockovpn](https://dockovpn.io) / Devops

### Source URL
https://github.com/dockovpn/dockovpn

### Why it is awesome
Dockovpn is an out of the box dockerized OpenVPN compatible server, which starts in just a few seconds and runs forever.

### Have you used it? For how long?
I've been sing it since 2019 and find it very helpful as do a lot of other Dockovpn users.

### Is this in a personal or professional setup?
I've ben using it for both: personal and professional setups. Personally, I use it every time I need to virtually place myself elswhere in the world using any virtual private server. For professional setup, we're curently building a cloud hosted solution, which it capable of managing thousands of simultaneous users.

### How many devices/users/services/... do you manage with it?
I was capable of running it in a setup with dozens of simultaneous users.

#### Pros
- Extremely easy to use
- Starts up in a few seconds
- Can be used in a stateless mode
- Code gets reviewed from cybersecurity professionals from all over the world
#### Cons
- As of now lacks support for Wireguard and IKEv2

### CODE REVIEW

**Review by KieranRobson — CHANGES_REQUESTED** [review]
https://github.com/awesome-foss/awesome-sysadmin/blob/master/.github/CONTRIBUTING.md

Please order your contribution alphabetically

**Review by KieranRobson — APPROVED** [review]

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -639,6 +639,7 @@ Comparison of NoSQL servers: http://kkovacs.eu/cassandra-vs-mongodb-vs-couchdb-v
 
 *VPN software.*
 
+* [Dockovpn](https://dockovpn.io) - Out of the box stateless dockerized openvpn compartible server which starts in less than 2 seconds.
```
[the remark this answers is not in the document]
Comment (alekslitvinenk) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -642,6 +642,7 @@ Comparison of NoSQL servers: http://kkovacs.eu/cassandra-vs-mongodb-vs-couchdb-v
 
 *VPN software.*
 
+* [Dockovpn](https://dockovpn.io) - Out-of-the-box stateless dockerized OpenVPN-compartible server which starts in less than 2 seconds.
```
Comment (xrat) [inline README.md:645]: s/compartible/compatible/

**Review by nodiscc — CHANGES_REQUESTED** [review]
Hi @alekslitvinenk , since https://github.com/awesome-foss/awesome-sysadmin/pull/459 we have new contribution guidelines in place, could you please take some time to answer the following questions as best you can? Thanks

- **Why is it awesome?**
- **Have you used it? For how long?**
- **Is this in a personal or professional setup?**
- **How many devices/users/services/... do you manage with it?**
- **Biggest pros/cons compared to other solutions?**

**Comment by alekslitvinenk** [comment]
@nodiscc I updated PR message

**Review by nodiscc — APPROVED** [review]

**Review by Technetium1 — APPROVED** [review]

**Pull request merged**